#VRML_SIM R2021a utf8
WorldInfo {
  basicTimeStep 3
  coordinateSystem "NUE"
  contactProperties [
    ContactProperties {
      material2 "slippy"
      coulombFriction [
        0
      ]
    }
    ContactProperties {
      material2 "block"
      coulombFriction [
        0.01
      ]
    }
  ]
}
Viewpoint {
  orientation -0.9979473947138711 -0.054479945368802386 0.03365906915542078 1.4816668139848452
  position -0.15817064872204178 5.361724299755276 0.48796068208142457
}
Background {
  skyColor [
    0.15 0.45 1
  ]
}
Floor {
  size 2.4 2.4
  appearance Asphalt {
  }
}
target {
  translation -0.5242 0.05 -0.144
  name "target(1)"
}
target {
  translation -1.0665 0.05 -0.2783
  name "target(1)"
}
Robot {
  children [
    Display {
      name "display_navigation"
      width 16
      height 16
    }
    Display {
      name "display_pathfinding"
      width 16
      height 16
    }
    Display {
      name "display_explored"
      width 120
      height 120
    }
    Display {
      name "display_occupancy"
      width 120
      height 120
    }
    Receiver {
      channel 1
    }
    Emitter {
      channel 2
    }
  ]
  name "robot(1)"
  controller "external_pc"
}
target {
  translation -0.945809 0.05 0.624
  name "target(2)"
}
target {
  translation 0.7601 0.05 0.6937
  name "target(1)"
}
target {
  translation 0.4578 0.05 -0.7358
  colour 0 1 0
  name "target(1)"
}
target {
  translation -0.4298 0.04 -0.0241
  colour 0 1 0
  name "target(0)"
}
target {
  translation 1.058 0.04 -0.1279
  colour 0 1 0
  name "target(3)"
}
target {
  translation -1.0941 0.04 0.3438
  colour 0 1 0
  name "target(1)"
}
DEF Floor_colour Solid {
  translation 0 0.0001 0.4
  children [
    Shape {
      appearance PBRAppearance {
        baseColor 1 0 0
      }
      geometry Plane {
        size 0.4 0.4
      }
    }
  ]
  name "solid(6)"
}
DEF Floor_colour Solid {
  translation 0 0.0001 -0.4
  children [
    Shape {
      appearance PBRAppearance {
        baseColor 0 1 0
      }
      geometry Plane {
        size 0.4 0.4
      }
    }
  ]
  name "solid(5)"
}
DEF WALL_1 Solid {
  translation 1.2 0.05 0
  children [
    DEF WALL_SHAPE Shape {
      appearance PBRAppearance {
        baseColor 0 0.333333 0
        roughness 1
        metalness 0
      }
      geometry Box {
        size 0.02 0.1 2.38
      }
    }
  ]
  boundingObject USE WALL_SHAPE
}
DEF WALL_2 Solid {
  translation -1.2 0.05 0
  children [
    USE WALL_SHAPE
  ]
  name "solid(1)"
  boundingObject USE WALL_SHAPE
}
DEF WALL_3 Solid {
  translation 0 0.05 -1.2
  rotation 0 1 0 1.5708
  children [
    USE WALL_SHAPE
  ]
  name "solid(2)"
  boundingObject USE WALL_SHAPE
}
DEF WALL_4 Solid {
  translation 0 0.05 1.2
  rotation 0 1 0 1.5708
  children [
    USE WALL_SHAPE
  ]
  name "solid(3)"
  boundingObject USE WALL_SHAPE
}
TexturedBackgroundLight {
  texture "empty_office"
}
small_or_fluffy {
  translation 0 0.1 0.39
}
small_or_fluffy {
  name "Fluffy"
  translation 0 0.1 -0.4
  color 0 1 0
}
